cytominer_eval/tests/test_transform/test_util.py

```diff
@@ -116,27 +116,37 @@ def test_check_replicate_groups():
     replicate_groups = ["Metadata_gene_name", "Metadata_pert_name"]
     replicate_group_dict = {"replicate_id": "testingA", "group_id": "testingB"}
     for operation in available_metrics:
-        if operation != "grit":
+        if operation == "grit":
             check_replicate_groups(
-                eval_metric=operation, replicate_groups=replicate_groups
+                eval_metric=operation, replicate_groups=replicate_group_dict
             )
             with pytest.raises(AssertionError) as ae:
                 output = check_replicate_groups(
-                    eval_metric=operation, replicate_groups=replicate_group_dict
+                    eval_metric=operation, replicate_groups=replicate_groups
                 )
-            assert "Replicate groups must be a list for the {op} operation".format(
-                op=operation
-            ) in str(ae.value)
-        else:
+            assert "For grit, replicate_groups must be a dict" in str(ae.value)
+        elif operation == "mp_value":
             check_replicate_groups(
-                eval_metric=operation, replicate_groups=replicate_group_dict
+                eval_metric=operation, replicate_groups=replicate_groups[0]
```

Comment:
Why are we using only `Metadata_gene_name` here for replicate groups? It is possible to have multiple columns indicating which perturbations are replicates. `mp_value` doesn't support that?

Reply:
Looking back at `calculate_mp_value()` it should be a quick fix to support a list of columns (it's just used in the groupby).

I looked back at [`grit()`](https://github.com/cytomining/cytominer-eval/blob/56bd9e545d4ce5dea8c2d3897024a4eb241d06db/cytominer_eval/operations/grit.py) and noticed that I  used the same strategy though, so maybe it is ok for now.

If we don't change it, we should add an issue documenting that this could be implemented as an enhancement in both `mp_value` and `grit`.

Reply:
I agree, I saw both and had issues making it work with multiple columns so I settled for the single column like for `grit()` for now. This could be added as an enhancement.

Reply:
cool, added in #28